From be68c39c9b1d15138ba208a14e9a473130b7c246 Mon Sep 17 00:00:00 2001
From: Split7fire <split7fire@yandex.ru>
Date: Sat, 15 Aug 2020 10:28:05 +0700
Subject: [PATCH] Decoration: Change device name to Lenovo S5

---
 phh/treble/generate.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/phh/treble/generate.sh b/phh/treble/generate.sh
index c347141..4f5330a 100644
--- a/phh/treble/generate.sh
+++ b/phh/treble/generate.sh
@@ -74,7 +74,7 @@ $rom_script
 PRODUCT_NAME := $target
 PRODUCT_DEVICE := phhgsi_${arch}_$part
 PRODUCT_BRAND := Android
-PRODUCT_MODEL := Phh-Treble $apps_name
+PRODUCT_MODEL := Lenovo S5
 
 PRODUCT_PACKAGES += $extra_packages
 EOF
-- 
2.17.1

